src/button.tsx

```diff
@@ -475,134 +477,163 @@ interface ButtonLinkToProps extends ButtonLinkCommonProps {
 
 export type ButtonLinkProps = ButtonLinkOnPressProps | ButtonLinkHrefProps | ButtonLinkToProps;
 
-export const ButtonLink = React.forwardRef<TouchableElement, ButtonLinkProps>((props, ref) => {
-    const {formStatus} = useForm();
-    const isInverse = useIsInverseVariant();
-    const {analytics, textPresets} = useTheme();
-
-    const {loadingText} = props;
-    const isFormSending = formStatus === 'sending';
-    const [isOnPressPromiseResolving, setIsOnPressPromiseResolving] = React.useState(false);
-
-    const showSpinner = props.showSpinner || isOnPressPromiseResolving;
-    const showChevron = props.withChevron ?? (!!props.href || !!props.to);
+export const BaseButtonLink = React.forwardRef<TouchableElement, ButtonLinkProps & {type: ButtonLinkType}>(
+    ({type, ...props}, ref) => {
+        const {formStatus} = useForm();
+        const isInverse = useIsInverseVariant();
+        const {textPresets} = useTheme();
+        const {eventFormat} = useTrackingConfig();
```

Comment:
Unrelated change: eventFormat should be imported by `useTrackingConfig()`, not from `useTheme()`